components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt

```diff
@@ -31,6 +32,16 @@ open class TabCounter @JvmOverloads constructor(
 
         setCount(INTERNAL_COUNT)
 
+        context.obtainStyledAttributes(attrs, R.styleable.TabCounter, defStyle, 0).apply {
```

Comment:
The `TabCounterMenu` in this package uses the same workaround. Should we apply this fix there as well? We can keep/file separately though.

Here's how it's used in Fenix:
https://github.com/mozilla-mobile/fenix/blob/f263b7333a31027986845d82d93fa07e3b740661/app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarIntegration.kt#L158

Reply:
Oh I didn't see that. Yeah I think we can do the same thing here as well.

Reply:
Hrm.. I spoke too soon. We can't do the same here because the Menu doesn't extend from an Android View, so there is no AttributeSet where we can resolve the colour from.

Reply:
Maybe this change isn't worth it if it means splitting up the APIs to resolve the colour in two different ways.

Reply:
Hmm don't have an opinion. Whichever you prefer.

Reply:
Alright, let's land it!